back/src/domains/convention/use-cases/notifications/NotifyIcUserAgencyRightChanged.unit.test.ts

```diff
@@ -99,7 +99,7 @@ describe("SendEmailWhenAgencyIsActivated", () => {
       },
       agencyRights: [
         {
-          roles: ["to-review"],
+          roles: ["counsellor"],
```

Comment:
Je comprends pas trop pourquoi les valeurs ont changés ici, je m'attendais à ce que le test de ce usecase ne change pas

Reply:
Le test a changé car avant on passait directement les roles dans le .execute() du usecase, alors que maintenant on va les chercher en DB. Donc il faut que les rôles en DB ne soit pas "in-review": on ne copie les valideurs de la structure prescriptrice parents que si l'utilisateur n'est pas "in-review".
Avant, ce test n'était pas cohérent.